gyp/pylib/gyp/easy_xml.py

```diff
@@ -121,7 +122,14 @@ def WriteXmlIfChanged(content, path, encoding="utf-8", pretty=False, win32=False
 
     default_encoding = locale.getdefaultlocale()[1]
     if default_encoding and default_encoding.upper() != encoding.upper():
-        xml_string = xml_string.encode(encoding)
+        if sys.platform == "win32":
+            if (sys.version_info[0] + sys.version_info[1] * 0.1) > 3.7:
+                xml_string = xml_string.encode(encoding)
+            else:
+                xml_string = xml_string.decode("cp1251").encode(encoding)
+        # for non windows systems
+        else:
+            xml_string = xml_string.encode(encoding)
```

Comment:
```suggestion
        if sys.platform == "win32" and sys.version_info < (3, 7):
            xml_string = xml_string.decode("cp1251").encode(encoding)
        else:
            xml_string = xml_string.encode(encoding)
```
Let's compare a tuple and a tuple https://docs.python.org/3/library/sys.html#sys.version_info